main.go

```diff
@@ -327,7 +382,7 @@ func main() {
 		}
 		sort.Strings(keys)
 		for _, key := range keys {
-			logger.Printf("Title: %s - Due Date: %s", key, newMilestones[key])
+			logger.Printf("Title: %s - Due Date: %s", key, newMilestones[key].DueDate)
```

Comment:
Probably more ideal would be to use `newMilestones[key].Title` instead of `key`. It makes the value clearer and the key might change in the future.